hack/release.sh

```diff
@@ -10,13 +10,13 @@ function main () {
   make devkit-arm64
   make devkit-amd64
```

Comment:
im wondering if we can get rid of these :thinking:

Reply:
we can remove it. The change will be part of https://github.com/mesosphere/konvoy-image-builder/pull/708 . should be ready today.

Reply:
nice!